src/app/api/auth/signup/route.ts

```diff
@@ -7,25 +7,21 @@ import { customerSignUpValidationSchema } from "@/lib/validation-schema";
 
 export async function POST(req: NextRequest) {
   try {
-    const { name, email, password, confirmPassword, phone } = await req.json();
-
-    if (!name?.trim()) {
```

Comment:
The model is already nullable for name, by deleting this, it stops checking for name to be required.